### PR TITLE
FWI-1524 - Support additionalLabels, additionalPodLables for all Deployments

### PR DESCRIPTION
**Pull request by jdesouza**
**Why This PR?**
_a short description of why this PR is needed_

Fixes #

**Changes**
Changes proposed in this pull request:

*
*

**Checklist:**

* [ ] I have included the name of the chart in the title of this PR in square brackets i.e. `[stable/goldilocks]`.
* [ ] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] Any new values have been added to the README for the Chart, or `helm-docs --sort-values-order=file` has been run for the charts that support it.

[Internal Ticket FWI-1524](https://fairwinds.myjetbrains.com/youtrack/issue/FWI-1524)

### CODE REVIEW

File: stable/goldilocks/values.yaml [hunk 1]
```diff
@@ -174,3 +174,7 @@ dashboard:
   affinity: {}
   # dashboard.topologySpreadConstraints -- Topology spread constraints for the dashboard pods
   topologySpreadConstraints: []
+
+deployment:
+  additionalLabels:
+  additionalPodLabels:
```
Comment (rbren) [inline stable/goldilocks/values.yaml]: I think this was supposed to be in stable/fairwinds-insights/values.yaml 😄

**Review by rbren — COMMENTED** [review]
Looks good! Two suggestions:
* can we add some labels to `stable/fairwinds-insights/ci/values.yaml`? That will help test it
* can we call the object `deployments` instead of `deployment`?

You mean ci/test-values.yaml right?

File: stable/goldilocks/values.yaml (hunk 1 above)
Comment (jdesouza) [reply stable/goldilocks/values.yaml]: Yea fixed that :-)

Comment (jdesouza) [reply stable/goldilocks/values.yaml]: done

**Review by rbren — APPROVED** [review]

**Pull request merged**